core/db/migrate/20221229132350_create_spree_google_feed_settings.rb

```diff
@@ -0,0 +1,16 @@
+class CreateSpreeGoogleFeedSettings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_google_feed_settings do |t|
+      t.references :spree_store
```

Comment:
The flag seems like a good idea. I'm not sure aboute general table. It would probably require moving the columns that are set to to true or false to a list, or a dict? Becouse right now file generation assumes that column names match attribute names and I'm not sure that those names are universal.